.cookiecutter.json

```
@@ -21,15 +21,15 @@
         "_drift_manager": {
             "template": "https://github.com/nautobot/cookiecutter-nautobot-app.git",
             "template_dir": "nautobot-app",
-            "template_ref": "refs/tags/nautobot-app-v2.2.1",
+            "template_ref": "nautobot-app-v2.3.0",
```

```suggestion
            "template_ref": "refs/tags/nautobot-app-v2.3.0",
```

https://github.com/nautobot/nautobot-app-ssot/pull/501/files#diff-f01e5e871f7007cc648450ef7989f8482bcfc67bafb28a37e318a6a3c1866f3c